model/src/main/kotlin/OrtResult.kt

```diff
@@ -211,18 +211,9 @@ data class OrtResult(
      */
     fun collectProjectsAndPackages(includeSubProjects: Boolean = true): Set<Identifier> {
         val projectsAndPackages = mutableSetOf<Identifier>()
-        val projects = getProjects()
+        val projects = getProjects(includeSubProjects)
```

Comment:
You need to use the named argument, otherwise you are using the `omitExcluded` parameter.

Reply:
Doh, thanks 😆